src/lowlevel/webusb.js

```diff
@@ -0,0 +1,209 @@
+/* @flow */
+
+declare var __VERSION__: string;
+
+import EventEmitter from 'events';
```

Comment:
switch to events/events.js or eventemitter3

Reply:
unused - removed